packages/examples/.storybook/main.js

```diff
@@ -1,90 +1,18 @@
 // @ts-check
-import HardSourceWebpackPlugin from 'hard-source-webpack-plugin';
-import IgnoreNotFoundExportWebpackPlugin from 'ignore-not-found-export-webpack-plugin';
+import custom from '@uifabric/build/storybook/webpack.config';
 import * as path from 'path';
 import * as webpack from 'webpack';
-import getResolveAlias from '@uifabric/build/webpack/getResolveAlias';
 
 export default {
   addons: ['@storybook/addon-a11y/register'],
   webpackFinal: (/** @type {webpack.Configuration} */ config) => {
-    config.resolveLoader = {
-      ...config.resolveLoader,
-      modules: [
-        'node_modules',
-        path.resolve(__dirname, '../../../scripts/node_modules'),
-        path.resolve(__dirname, '../../../node_modules'),
-      ],
-    };
+    config = custom({ config, cwd: path.dirname(__dirname) });
```

Comment:
The custom setup that was previously here seemed mostly redundant with the shared webpack config.

cwd is used to ensure that aliases for all deps of the examples package (not just deps of the package being demo'd) are included.